frontend/public/components/routes.tsx

```diff
@@ -267,6 +222,25 @@ const RouteTargetRow = ({route, target}) => <tr>
   <td>{calcTrafficPercentage(target.weight, route)}</td>
 </tr>;
 
+const IngressDetails: React.SFC<IngressDetailsProps> = ({ingresses}) => <React.Fragment>
```

Comment:
You shouldn't need the outer React.Fragment. It can just be

```jsx
const IngressDetails: React.SFC<IngressDetailsProps> = ({ingresses}) => _.map(ingresses, ...);
```

Reply:
I'd call this `RouteIngressStatus`

Reply:
When I do: `const IngressDetails: React.SFC<IngressDetailsProps> = ({ingresses}) => _.map(..`, I get:
```
ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/public/components/routes.tsx(225,7):
TS2322: Type '({ ingresses }: RouteIngressStatusProps & { children?: ReactNode; }) => Element[]' is not assignable to type 'StatelessComponent<RouteIngressStatusProps>'.
  Type 'Element[]' is not assignable to type 'ReactElement<any>'.
    Property 'type' is missing in type 'Element[]'.
```
Component:
```
const RouteIngressStatus: React.SFC<RouteIngressStatusProps> = ({ingresses}) => _.map(ingresses, (ingress, i) =>
  <React.Fragment key={i}>
    <h1 className="co-section-title">{ingress.host}</h1>
    <dl>
      <dt>Router</dt>
      <dd>{ingress.routerName}</dd>
      <dt>Wildcard Policy</dt>
      <dd>{ingress.wildcardPolicy}</dd>
    </dl>
    <h4 className="co-m-route-ingress-conditions___title">Conditions</h4>
    <Conditions key={ingress.host} conditions={ingress.conditions} />
  </React.Fragment>);
```
I think because whatever is returned needs to have a single root parent element; the `_.map` returns an array of children (multiple ingress status tables...if they exist)